oauth2/oauth2-jose/src/test/java/org/springframework/security/oauth2/jwt/MappedJwtClaimSetConverterTests.java

```diff
@@ -140,30 +138,6 @@ public void convertWhenUsingCustomConverterThenAllOtherDefaultsAreStillUsed() th
 		assertThat(target.get(JwtClaimNames.SUB)).isEqualTo("1234");
 	}
 
-	@Test
-	public void convertWhenConverterReturnsNullThenClaimIsRemoved() {
-		MappedJwtClaimSetConverter converter = MappedJwtClaimSetConverter
-				.withDefaults(Collections.emptyMap());
-
-		Map<String, Object> source = Collections.singletonMap(JwtClaimNames.ISS, null);
-		Map<String, Object> target = converter.convert(source);
-
-		assertThat(target).doesNotContainKey(JwtClaimNames.ISS);
-	}
-
-	@Test
-	public void convertWhenConverterReturnsValueWhenEntryIsMissingThenEntryIsAdded() {
```

Comment:
@jzheaux I removed this test as described in #6800

Reply:
I'm not keen on removing these test until we come to an agreement on #6800. This PR isn't about changing how `MappedJwtClaimSetConverter` behaves.